Utilities/StorageFactory/test/BuildFile.xml

```diff
@@ -1,39 +1,18 @@
-<use   name="Utilities/StorageFactory"/>
-<use   name="FWCore/ParameterSet"/>
-<bin   file="t0Repack.cpp" name="test_StorageFactory_t0Repack">
-  <flags NO_TESTRUN="1"/>
-</bin>
-<bin   file="randomread.cpp" name="test_StorageFactory_randomread">
-  <flags NO_TESTRUN="1"/>
-</bin>
-<bin   file="any.cpp" name="test_StorageFactory_Any">
-  <flags NO_TESTRUN="1"/>
-</bin>
-<bin   file="local.cpp" name="test_StorageFactory_Local">
-</bin>
-<bin   file="local2.cpp" name="test_StorageFactory_Local2">
-</bin>
-<bin   file="local3.cpp" name="test_StorageFactory_Local3">
-</bin>
-<bin   file="ftp.cpp" name="test_StorageFactory_Ftp">
-  <flags NO_TESTRUN="1"/>
-</bin>
-<bin   file="ftp2.cpp" name="test_StorageFactory_Ftp2">
-  <flags NO_TESTRUN="1"/>
-</bin>
-<bin   file="rfio2.cpp" name="test_StorageFactory_Rfio2">
-</bin>
-<bin   file="rfio3.cpp" name="test_StorageFactory_Rfio3">
-</bin>
-<bin   file="write.cpp" name="test_StorageFactory_Write">
-  <flags NO_TESTRUN="1"/>
-</bin>
-<bin   file="mkstemp.cpp" name="test_StorageFactory_Mkstemp">
+</bin><bin file="local2.cpp" name="test_StorageFactory_Local2">
```

Comment:
This does not look correct. @Dr15Jones

Reply:
Good catch! I'm guessing this is caused by the fact that `BuildFile.xml` files in general are not actually fully XML conformant (i.e. XML requires one and only one top level node while the `BuildFile.xml` format does not.)

Reply:
The comment says

> we want to wait until the framework decides on a threading model to implement a fix

maybe it's obsolete ?

Reply:
> The comment says
> 
> > we want to wait until the framework decides on a threading model to implement a fix
> 
> maybe it's obsolete ?

The test was disabled in https://github.com/cms-sw/cmssw/commit/e09f0ac72480f91cc995fb8e1fcdc39bfbd79ac7. @Dr15Jones Would you be able to tell if there is anything that would prevent the test to run?